### PR TITLE
added german count

### PR DESCRIPTION
**Pull request by george-gino**
added a random so that it does either guten morgen or count german

### CODE REVIEW

**Review by AmoleR — CHANGES_REQUESTED** [review]
In general, here are a few tips.
- Store random variables in an `ArrayList` so that you can just access it with `array.get(random)`, and adding more is easy.
- Delete extra lines (i.e. line 21).
- Don't use `{ .. }` for one line things, do
```java
if (check == 1)
  sendMessage(...);
else
  sendMessage(...);
```

File: src/main/java/Listener.java [hunk 1]
```diff
@@ -17,9 +17,8 @@ private void conlin(MessageReceivedEvent event) {
             "> The limit of the function never depends on the value of the function at that point.\n\n- John Conlin"
         );
     }
-    private void german(MessageReceivedEvent event){
-        sendMessage(event, "Guten Morgen");
-        int check = (int)(Math.random() * (2 - 1 + 1) + 1);
+    private void foreign(MessageReceivedEvent event){
+        int check = (int)(Math.random() * (4 - 1 + 1) + 1);
```
Comment (VincentZhao12) [inline src/main/java/Listener.java:21]: -1+1 is pointless

File: src/main/java/Listener.java [hunk 2]
```diff
@@ -214,8 +222,8 @@ public void onMessageReceived(MessageReceivedEvent event) {
             case "math-help":
                 mathHelp(event);
                 break;
-            case "german";
-                german(event);
+            case "foreign";
+                foreign(event);
```
Comment (VincentZhao12) [inline src/main/java/Listener.java:226]: do u know how to write a case for switch case? do u not see the other cases above? can u not copy paste?

**Review by VincentZhao12 — COMMENTED** [review]
please learn java

**Pull request closed without merging**